src/library_webgpu.js

```diff
@@ -2640,6 +2647,8 @@ var LibraryWebGPU = {
       var deviceLostCallbackPtr = {{{ makeGetValue('descriptor', C_STRUCTS.WGPUDeviceDescriptor.deviceLostCallback, '*') }}};
       var deviceLostUserdataPtr = {{{ makeGetValue('descriptor', C_STRUCTS.WGPUDeviceDescriptor.deviceLostUserdata, '*') }}};
 
+      var uncapturedErrorCallbackInfoPtr = {{{ makeGetValue('descriptor', C_STRUCTS.WGPUDeviceDescriptor.uncapturedErrorCallbackInfo, '*') }}};
```

Comment:
@kainino0x Do you have any idea why `uncapturedErrorCallbackInfoPtr` is always 0 when I run `webgpu_basic_rendering.cpp` test? It looks like I've missed adding something but I can't figure it out.

Reply:
`uncapturedErrorCallbackInfo` isn't a pointer, it's a struct. So (in 32 bit) you're pulling the first 4 bytes of the uncapturedErrorCallbackInfo, which is `nextInChain`, which is `nullptr`.

Instead you need something like `C_STRUCTS.WGPUDeviceDescriptor.uncapturedErrorCallbackInfo + C_STRUCTS.WGPUUncapturedErrorCallbackInfo.callback`.

Reply:
Thank you @kainino0x!